### PR TITLE
skin/confluence: Minor relocations of elements on fullscreen osd

### PR DESCRIPTION
**Pull request by nemphys**
Sorry, just noticed that a few adjustments where needed for better eye-candy

### CODE REVIEW

**Comment by opdenkamp** [comment]
alright, that one's better indeed.

got some more gui changes planned? cause I'll keep this pull req open in that case and merge when you're done, to minimise the number of merge commits that are not needed.

**Comment by nemphys** [comment]
Fair enough, I'll check if I have any other changes tomorrow and inform you as soon as I'm done, thanx

**Comment by nemphys** [comment]
Check this out, it looks cool

File: xbmc/GUIInfoManager.h [hunk 1]
```diff
@@ -446,9 +446,11 @@ class CDateTime;
 #define PVR_ACTUAL_STREAM_ENCRYPTED 1110
 
 #define PVR_NEXT_RECORDING_CHANNEL  1120
+#define PVR_NEXT_RECORDING_CHAN_ICO 1118
```
Comment (nemphys) [inline xbmc/GUIInfoManager.h:449]: Not sure about this one, though. The numbering had no spaces, so I went with this

File: xbmc/GUIInfoManager.h [hunk 2]
```diff
@@ -446,9 +446,11 @@ class CDateTime;
...
 #define PVR_NEXT_RECORDING_DATETIME 1121
 #define PVR_NEXT_RECORDING_TITLE    1122
 #define PVR_NOW_RECORDING_CHANNEL   1123
+#define PVR_NOW_RECORDING_CHAN_ICO  1119
```
Comment (nemphys) [inline xbmc/GUIInfoManager.h:453]: Same here

**Comment by nemphys** [comment]
Any comments on this one?

**Comment by opdenkamp** [comment]
didn't check it out yet. and likely won't tomorrow either because I won't be sitting behind a keyboard then (national celebration here tomorrow)

**Comment by opdenkamp** [comment]
right, checked it out this morning and it looks fine. thanks.
just one issue I found: when a recording is running on a channel and you start playback on that channel and press M, you can see all the recording icons, but the recording button is not visible.

**Comment by nemphys** [comment]
Right, I didn't see this because it does not happen here. What I just realized is that, since I renamed OSDRecordNF2.png to OSDRecord2NF.png (for naming consistency), git picked the deletion of the former, but did not get the addition of the latter. Maybe because it's a binary file and I'm totally new to git :-) 
Any ideas about how to include the file?

**Comment by opdenkamp** [comment]
ah right. I only checked for references to OSDRecordNF2 before accepting that deletion, assuming there was a file without 2 at the end :)

I'll fix it. next time: "git rm /path/to/oldfile" and "git add /path/to/newfile"

**Comment by nemphys** [comment]
No, I'll make a new pull request along with another minor fix, since I also make a few changes to the actual image

**Comment by opdenkamp** [comment]
commit 5b6549afb4ef247f30386db836b71ba506f3cbd4 has been reverted in bdd56ae707b1d2b0e8f7504d115fafbfac481b7c by Jezz_X. an explanation can be found in the commit message.

**Pull request merged**